tests/bears/requirements/RscriptRequirementTest.py

```diff
@@ -1,7 +1,7 @@
 import shutil
 import unittest
 from coalib.bears.requirements.RscriptRequirement import RscriptRequirement
-from coalib.misc.Shell import call_without_output
+from coalib.misc.Shell import call_without_output  # Ignore PyUnusedCodeBear
```

Comment:
I don't think we need this either... It's just a test. Remove and see if any of the tests break....